src/main/webapp/js/instructorFeedbackEdit/customFeedbackPaths.js

```diff
@@ -1,368 +1,379 @@
+        }
+    },
     
-    var isChangingParticipantTypeFromCustomToPredefined =
-            giverType === FEEDBACK_PARTICIPANT_TYPE_CUSTOM
-            || recipientType === FEEDBACK_PARTICIPANT_TYPE_CUSTOM;
-    if (isChangingParticipantTypeFromCustomToPredefined) {
-        $giverSelect.find('option[value="' + FEEDBACK_PARTICIPANT_TYPE_CUSTOM + '"]').remove();
-        $recipientSelect.find('option[value="' + FEEDBACK_PARTICIPANT_TYPE_CUSTOM + '"]').remove();
-    }
-}
-
-function appendCustomOptionsIfNecessary($questionForm) {
-    $questionForm.find('.participantSelect').each(function() {
-        if ($(this).val() !== FEEDBACK_PARTICIPANT_TYPE_CUSTOM) {
-            appendCustomOptionToParticipantSelect($(this));
+    appendCustomOptionsIfNecessary: function($questionForm) {
+        $questionForm.find('.participantSelect').each(function() {
+            if ($(this).val() !== customFeedbackPaths.FEEDBACK_PARTICIPANT_TYPE_CUSTOM) {
+                customFeedbackPaths.appendCustomOptionToParticipantSelect($(this));
+            }
+        });
+    },
+    
+    appendCustomOptionToParticipantSelect: function($participantSelect) {
+        var $customFeedbackParticipantTypeOption =
+                $('<option></option>').attr('value', customFeedbackPaths.FEEDBACK_PARTICIPANT_TYPE_CUSTOM)
+                                      .text('Custom');
+        $participantSelect.append($customFeedbackParticipantTypeOption)
+                          .val(customFeedbackPaths.FEEDBACK_PARTICIPANT_TYPE_CUSTOM);
+    },
+    
+    addRowsToFeedbackPathsSpreadsheet: function($questionForm) {
+        var $container = $questionForm.find('.custom-feedback-paths-spreadsheet');
+        var numRowsToAdd = parseInt($questionForm.find('.add-rows-input').val());
+        var hotInstance = $container.handsontable('getInstance');
+        if (numRowsToAdd > 0) {
+            hotInstance.alter('insert_row', hotInstance.countRows(), numRowsToAdd);
         }
-    });
-}
-
-function appendCustomOptionToParticipantSelect($participantSelect) {
-    var $customFeedbackParticipantTypeOption =
-            $('<option></option>').attr('value', FEEDBACK_PARTICIPANT_TYPE_CUSTOM)
-                                  .text('Custom');
-    $participantSelect.append($customFeedbackParticipantTypeOption)
-                      .val(FEEDBACK_PARTICIPANT_TYPE_CUSTOM);
-}
-
-function addRowsToFeedbackPathsSpreadsheet($questionForm) {
-    var $container = $questionForm.find('.custom-feedback-paths-spreadsheet');
-    var numRowsToAdd = parseInt($questionForm.find('.add-rows-input').val());
-    var hotInstance = $container.handsontable('getInstance');
-    if (numRowsToAdd > 0) {
-        hotInstance.alter('insert_row', hotInstance.countRows(), numRowsToAdd);
     }
-}
+};
+
+$(document).ready(function() {
+    customFeedbackPaths.initializeCustomFeedbackPathsData();
+    customFeedbackPaths.initializeFeedbackPathsSpreadsheets();
+    customFeedbackPaths.bindEventHandlers();
```

Comment:
as @wkurniawan07 suggested in https://github.com/TEAMMATES/teammates/pull/5744#discussion_r70209542, maybe put this in `instructorFeedbackEdit.js` so this `customFeedbackPaths` can be unit tested without needing `$(document).ready()`